packages/@o3r/application/schemas/functional-content.metadat.schema.json

```diff
@@ -0,0 +1,306 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "FunctionalContentsMetadataSchema",
+  "type": "object",
+  "title": "Schema of Functional Contents metadata",
+  "description": "Schema used to help to validate the functional contents metadata",
+  "required": [
+    "name",
+    "title",
+    "description",
+    "sections"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": " Id of the functional data linked to the metadata file. It has to be unique among all the functional contents metadata files"
+    },
+    "title": {
+      "type": "string",
+      "description": "Title to display to the CMS user"
+    },
+    "description": {
+      "type": "string",
+      "description": "Description to display to the CMS user"
+    },
+    "sections": {
+      "type": "array",
+      "description": "Array of functional content sections",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "description": "Functional content section",
+        "required": [
+          "name",
+          "title",
+          "description",
+          "fileName",
+          "rootReference",
+          "definitions"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Unique id of the section"
+          },
+          "title": {
+            "type": "string",
+            "description": "Title to display to the CMS user"
+          },
+          "description": {
+            "type": "string",
+            "description": "Description of the section"
+          },
+          "fileName": {
+            "type": "string",
+            "description": "Name of the file to use by the CMS to export the associated data. If the fileName is referenced in another section, the CMS will merge the data in the same exported file",
+            "pattern": ".*\\.json$"
+          },
+          "rootReference": {
+            "type": "string",
+            "description": "Name of the object definition to use as the root of the model described by this section"
+          },
+          "definitions": {
+            "type": "array",
+            "description": "Object definitions for this section",
+            "minItems": 1,
```

Comment:
why this `minItems: 1` to a mandatory list?

Reply:
The objective here is to have a mandatory list and it should not be empty.
Without setting the minItems, "sections":[] should not be allowed

Reply:
Not having definitions is not possible?